test/events/objectEvent.spec.js

```diff
@@ -19,6 +19,7 @@ import DestinationElement from '../../src/entity/model/DestinationElement';
 import SensorElement from '../../src/entity/model/sensor/SensorElement';
 import Ilmd from '../../src/entity/model/Ilmd';
 import { exampleObjectEvent } from '../data/eventExample';
+import { getTimeZoneOffsetFromStringOrNumber } from '../../src/utils/utils';
```

Comment:
`FromStringOrNumber` is evident from the inputs, so could shorten the name of this function.